Nightwatch/Configuration.fs

```diff
@@ -62,16 +57,22 @@ let private buildDeserializer() =
         .WithTypeConverter(versionConverter)
         .Build()
 
-let private toResource : Result<ResourceDescription, _> -> _ =
-    Result.map (fun res -> { id = res.id; runEvery = res.schedule; checkCommand = res.check })
+let private toResource (registry : ResourceRegistry) : Result<ResourceDescription * Path, _> -> _ =
```

Comment:
Very strange and a bit cryptic signature. Why is it semi-point-free? Better without explicit signature?

Reply:
Because why the hell not? I tend to automatically remove the trailing arguments from the functions most of the time.

See, here we're using `registry` argument and it's definitely _hard_ to remove that argument.

On the other hand, have I added the `resource` argument, its' only function would to stay at the end of the argument list, and be passed to the end of the resulting expression, like that:

```
let private toResource (...) resource = (...) resource
```

That's an easy-recognizable pattern I tend to simplify on sight.

Having said that, I think you're right blaming this definition: it's too long. We could strip the types here entirely (and that's another think I like to do in private functions: make the compiler to infer all the possible types). I'll remove the definition, thanks.

Reply:
I'm blaming it because of `_> -> _`. It's not I expect to see in the code. So there are two options: remove signature or avoid point-free and function type in the signature.

Reply:
As you can see, I've removed that!